randovania/games/samus_returns/logic_database/header.json

```diff
@@ -6058,10 +6062,10 @@
             ]
         },
         "baby": {
-            "long_name": "Baby Crystals",
+            "long_name": "Crystal Blocks",
```

Comment:
I only now realized this: do you think there could be confusion with a4-crystal mines?

Reply:
Possibly? Worth noting that the various DB comments just refer to them as Baby Blocks. Technically the official name is Crystals but that could cause confusion.

Reply:
I'll rename this one to be "familiar"